whitesource.config

```diff
@@ -0,0 +1,3 @@
+# Reference: https://whitesource.atlassian.net/wiki/spaces/WD/pages/1544880156/Unified+Agent+Configuration+Parameters
+
+gradle.ignoredConfigurations=.*tests.*
```

Comment:
I don't think this is useful for Python BDK. You can see parameters, per language, here: https://whitesource.atlassian.net/wiki/spaces/WD/pages/1544880156/Unified+Agent+Configuration+Parameters